release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -202,6 +202,18 @@ For more information, see _Managing machines with the Cluster API_.
 [id="ocp-4-16-nodes_{context}"]
 === Nodes
 
+[id="ocp-4-16-must-gather-directory_{context}"]
+==== Information added to directory when using must-gather
+
+This release collects the following additional information when you run the `oc adm must-gather` command:
```

Comment:
The release collects that information, or the application collects the information? 

Is this more accurate?: 
"The `oc-adm must-gather` command now collects the following information:"